### PR TITLE
feat(core/inlines): process code inside inline links

### PR DESCRIPTION
**Pull request by marcoscaceres**
Can now handle, for example, [= `link` element =]

### CODE REVIEW

File: src/core/inlines.js [hunk 1]
```diff
@@ -121,14 +126,27 @@ function inlineLinkMatches(matched) {
...
+function processInlineContent(text) {
+  if (inlineCodeRegExp.test(text)) {
+    // We use a capture group to split, so we can process all the parts.
+    return text.split(/(`[^`]+`)(?!`)/).map(part => {
+      return part.startsWith("`")
+        ? inlineCodeMatches(part)
+        : processInlineContent(part);
+    });
+  }
+  return hyperHTML`${text}`;
```
Comment (saschanaz) [inline src/core/inlines.js]: Maybe just `text`?

Comment (marcoscaceres) [reply src/core/inlines.js]: Just text confuses hyper, in that it then coerces all other items in the array to text.

Comment (saschanaz) [reply src/core/inlines.js]: Uhm, what array do you mean?

Comment (saschanaz) [reply src/core/inlines.js]: Ah okay, this is a recursive function.

Comment (saschanaz) [reply src/core/inlines.js]: But maybe `document.createTextNode(text)`? I think we prefered it on other parts of the code.

Comment (marcoscaceres) [reply src/core/inlines.js]: good suggestion.

**Pull request merged**